<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="tests.WebQuizEngineTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2020-05-01T17:54:17" hostname="home" time="6.809">
  <properties/>
  <testcase name="start" classname="tests.WebQuizEngineTest" time="6.809">
    <failure message="java.lang.AssertionError: Wrong answer in test #11&#10;&#10;The quiz sent to the program looked like this:&#10;{&#10;  &quot;title&quot;: &quot;The Java Logo&quot;,&#10;  &quot;text&quot;: &quot;What is depicted on the Java logo?&quot;,&#10;  &quot;options&quot;: [&#10;    &quot;Robot&quot;,&#10;    &quot;Tea leaf&quot;,&#10;    &quot;Cup of coffee&quot;,&#10;    &quot;Bug&quot;&#10;  ],&#10;  &quot;id&quot;: 3&#10;}&#10;&#10;But the received quiz looks like that:&#10;{&#10;  &quot;id&quot;: 3,&#10;  &quot;title&quot;: &quot;The Java Logo&quot;,&#10;  &quot;text&quot;: &quot;What is depicted on the Java logo?&quot;,&#10;  &quot;options&quot;: [&#10;    &quot;Robot&quot;,&#10;    &quot;Tea leaf&quot;,&#10;    &quot;Cup of coffee&quot;,&#10;    &quot;Bug&quot;&#10;  ]&#10;}" type="java.lang.AssertionError">java.lang.AssertionError: Wrong answer in test #11

The quiz sent to the program looked like this:
{
  &quot;title&quot;: &quot;The Java Logo&quot;,
  &quot;text&quot;: &quot;What is depicted on the Java logo?&quot;,
  &quot;options&quot;: [
    &quot;Robot&quot;,
    &quot;Tea leaf&quot;,
    &quot;Cup of coffee&quot;,
    &quot;Bug&quot;
  ],
  &quot;id&quot;: 3
}

But the received quiz looks like that:
{
  &quot;id&quot;: 3,
  &quot;title&quot;: &quot;The Java Logo&quot;,
  &quot;text&quot;: &quot;What is depicted on the Java logo?&quot;,
  &quot;options&quot;: [
    &quot;Robot&quot;,
    &quot;Tea leaf&quot;,
    &quot;Cup of coffee&quot;,
    &quot;Bug&quot;
  ]
}
	at org.junit.Assert.fail(Assert.java:88)
	at org.hyperskill.hstest.stage.StageTest.start(StageTest.java:102)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:118)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:175)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:157)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.base/java.lang.Thread.run(Thread.java:834)
</failure>
  </testcase>
  <system-out><![CDATA[?[1;31m
Start test 1?[0m

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v2.2.2.RELEASE)

2020-05-01 14:54:17.644  INFO 6548 --- [    Test worker] engine.WebQuizEngine                     : Starting WebQuizEngine on home with PID 6548 (started by rafae in C:\Projects\GitHub\jetbrains-academy\Web Quiz Engine\Web Quiz Engine\task)
2020-05-01 14:54:17.648  INFO 6548 --- [    Test worker] engine.WebQuizEngine                     : No active profile set, falling back to default profiles: default
2020-05-01 14:54:18.655  INFO 6548 --- [    Test worker] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-05-01 14:54:18.717  INFO 6548 --- [    Test worker] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 54ms. Found 2 JPA repository interfaces.
2020-05-01 14:54:19.110  INFO 6548 --- [    Test worker] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration' of type [org.springframework.transaction.annotation.ProxyTransactionManagementConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-05-01 14:54:19.406  INFO 6548 --- [    Test worker] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8889 (http)
2020-05-01 14:54:19.418  INFO 6548 --- [    Test worker] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-05-01 14:54:19.418  INFO 6548 --- [    Test worker] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.29]
2020-05-01 14:54:19.596  INFO 6548 --- [    Test worker] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-05-01 14:54:19.596  INFO 6548 --- [    Test worker] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 1896 ms
2020-05-01 14:54:19.726  INFO 6548 --- [    Test worker] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2020-05-01 14:54:19.960  INFO 6548 --- [    Test worker] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2020-05-01 14:54:19.968  INFO 6548 --- [    Test worker] o.s.b.a.h2.H2ConsoleAutoConfiguration    : H2 console available at '/h2-console'. Database available at 'jdbc:h2:file:../quizdb'
2020-05-01 14:54:20.245  INFO 6548 --- [    Test worker] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-05-01 14:54:20.361  INFO 6548 --- [    Test worker] org.hibernate.Version                    : HHH000412: Hibernate Core {5.4.9.Final}
2020-05-01 14:54:20.570  INFO 6548 --- [    Test worker] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-05-01 14:54:20.723  INFO 6548 --- [    Test worker] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.H2Dialect
2020-05-01 14:54:21.634  INFO 6548 --- [    Test worker] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-05-01 14:54:21.641  INFO 6548 --- [    Test worker] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-05-01 14:54:22.054  WARN 6548 --- [    Test worker] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-05-01 14:54:22.272  INFO 6548 --- [    Test worker] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@715413f2, org.springframework.security.web.context.SecurityContextPersistenceFilter@1667c17e, org.springframework.security.web.header.HeaderWriterFilter@2594a1fc, org.springframework.security.web.authentication.logout.LogoutFilter@41cb71a0, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@293df4f7, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@68cb25c1, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@68d9dada, org.springframework.security.web.session.SessionManagementFilter@661f105a, org.springframework.security.web.access.ExceptionTranslationFilter@4c19cd2e, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@1ee0d604]
2020-05-01 14:54:22.400  INFO 6548 --- [    Test worker] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2020-05-01 14:54:22.673  INFO 6548 --- [    Test worker] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 14 endpoint(s) beneath base path '/actuator'
2020-05-01 14:54:22.729  INFO 6548 --- [    Test worker] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8889 (http) with context path ''
2020-05-01 14:54:22.731  INFO 6548 --- [    Test worker] engine.WebQuizEngine                     : Started WebQuizEngine in 5.506 seconds (JVM running for 6.508)
2020-05-01 14:54:23.075  INFO 6548 --- [nio-8889-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-05-01 14:54:23.077  INFO 6548 --- [nio-8889-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-05-01 14:54:23.086  INFO 6548 --- [nio-8889-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 9 ms
?[1;31m
Start test 2?[0m
?[1;31m
Start test 3?[0m
2020-05-01 14:54:23.634  WARN 6548 --- [nio-8889-exec-4] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<engine.models.User> engine.controllers.UserController.post(engine.models.User): [Field error in object 'user' on field 'email': rejected value [google.com]; codes [Email.user.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@11bab547,.*\..*]; default message [must be a well-formed email address]] ]
?[1;31m
Start test 4?[0m
2020-05-01 14:54:23.670  WARN 6548 --- [nio-8889-exec-5] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<engine.models.User> engine.controllers.UserController.post(engine.models.User): [Field error in object 'user' on field 'email': rejected value [well@googlecom]; codes [Email.user.email,Email.email,Email.java.lang.String,Email]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@11bab547,.*\..*]; default message [must be a well-formed email address]] ]
?[1;31m
Start test 5?[0m
?[1;31m
Start test 6?[0m
2020-05-01 14:54:23.691  WARN 6548 --- [nio-8889-exec-7] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<engine.models.User> engine.controllers.UserController.post(engine.models.User): [Field error in object 'user' on field 'password': rejected value [1234]; codes [Size.user.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.password,password]; arguments []; default message [password],2147483647,5]; default message [size must be between 5 and 2147483647]] ]
?[1;31m
Start test 7?[0m
2020-05-01 14:54:23.699  WARN 6548 --- [nio-8889-exec-8] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<engine.models.User> engine.controllers.UserController.post(engine.models.User): [Field error in object 'user' on field 'password': rejected value [123]; codes [Size.user.password,Size.password,Size.java.lang.String,Size]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.password,password]; arguments []; default message [password],2147483647,5]; default message [size must be between 5 and 2147483647]] ]
?[1;31m
Start test 8?[0m
?[1;31m
Start test 9?[0m
?[1;31m
Start test 10?[0m
?[1;31m
Start test 11?[0m
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
